docs/PIPELINES-CONFIG.md

```diff
@@ -44,9 +44,169 @@ cat resources/tekton-config.json | kubectl patch tektonconfig config --type 'mer
+          echo "OK"
+        fi
+
+        SECRET_NAME="gitops-auth-secret"
+        if [ -n "$GIT_TOKEN" ]; then
+          echo -n "* $SECRET_NAME secret: "
+          kubectl create secret generic "$SECRET_NAME" \
+            --from-literal=password=$GIT_TOKEN \
+            --type=kubernetes.io/basic-auth \
+            --dry-run=client -o yaml | kubectl apply --filename - --overwrite=true >/dev/null
+          echo "OK"
+        fi
+
+        SECRET_NAME="pipelines-secret"
+        if [ -n "$PIPELINES_WEBHOOK_SECRET" ]; then
+          echo -n "* $SECRET_NAME secret: "
+          kubectl create secret generic "$SECRET_NAME" \
+            --from-literal=webhook.secret=$PIPELINES_WEBHOOK_SECRET \
+            --dry-run=client -o yaml | kubectl apply --filename - --overwrite=true >/dev/null
+          echo "OK"
+        fi
+
+        SECRET_NAME="rhdh-image-registry-token"
+        if [ -n "$QUAY_DOCKERCONFIGJSON" ]; then
+          echo -n "* $SECRET_NAME secret: "
+          DATA=$(mktemp)
+          echo -n "$QUAY_DOCKERCONFIGJSON" >"$DATA"
+          kubectl create secret docker-registry "$SECRET_NAME" \
+            --from-file=.dockerconfigjson="$DATA" --dry-run=client -o yaml | \
+            kubectl apply --filename - --overwrite=true >/dev/null
+          rm "$DATA"
+          echo -n "."
+          while ! kubectl get serviceaccount pipeline >/dev/null &>2; do
+            sleep 2
+            echo -n "_"
+          done
+          for SA in default pipeline; do
+            kubectl patch serviceaccounts "$SA" --patch "
+          secrets:
+            - name: $SECRET_NAME
+          imagePullSecrets:
+            - name: $SECRET_NAME
+          " >/dev/null
+            echo -n "."
+          done
+          echo "OK"
+        fi"
+```
+
+See the following optional steps for more information about each secret being created in this Task:
+1. [Setting Up Gitops Authentication Secret Under Deployment Namespaces](#step-6-setting-up-gitops-authentication-secret-under-deployment-namespaces-optional)
+2. [Setting Up Pipelines Secret Under Deployment Namespaces](#step-7-setting-up-pipelines-secret-under-deployment-namespaces-optional)
+3. [Setting Up Quay Image Registry Secret Under Deployment Namespaces](#step-8-setting-up-quay-image-registry-secret-under-deployment-namespaces-optional)
+
+#### Step 4: Setting Up Deployment Namespaces \(Optional\)
+
+Optionally, you can create the namespaces for the different app deployments, this will mirror some the steps for the Tekton Task created. You will need a namespace for each kind of the deployment: development, staging, and production. These namespaces would follow a naming pattern of `<rhdh_namepsace>-app-<deployment_kind>` with the target RHDH namespace as the prefix, therefore `$NAMESPACE-app-development`, `$NAMESPACE-app-stage`, and `$NAMESPACE-app-prod`. These namespaces can be created using the following:
```

Comment:
@Jdubrick Yeah I would opt out of making this change this late since it would affect the technical part of how it works. Also, based on prior discussions this part is planned to be removed anyway.